hadoop-hdds/docs/content/start/FromSource.md

```diff
@@ -25,6 +25,7 @@ weight: 30
 * JDK 1.8 or higher
 * [Maven 3.6 or later](https://maven.apache.org/download.cgi)
 * Internet connection for first build (to fetch all Maven and Ozone dependencies)
+* [Docker Compose v2.32.1 or higher](https://docs.docker.com/compose/install/) (to prevent unexpected character “-” in variable name issue)
```

Comment:
According to https://issues.apache.org/jira/browse/HDDS-11590
Docker Compose v2.31 or above works.

Reply:
Also, I don't think we need to mention the specific problem.

```suggestion
* [Docker Compose v2.30 or newer](https://docs.docker.com/compose/install/)
```

Reply:
Got it, thanks @adoroszlai.

I push the new commit for that, thanks!